### DISCUSSION TITLE
How does the "Inconclusive" score (-1) impact the overall score?

### DISCUSSION THREAD

**Discussion by ajmalab** [ossf/scorecard#3261] [Q&A]
Hey all, couldn't find this anywhere in the docs but, how does an inconclusive score impact the aggregate score? Is the -1 ignored or included in the aggregate score?

**Comment by spencerschrock** [comment]
You can see the logic here:
https://github.com/ossf/scorecard/blob/397c97f766bb6c2cf6740c698e52e5c74de40dbb/pkg/scorecard_result.go#L64-L65

Specifically here, which ignores inconclusive results (-1) when calculating the aggregate:
https://github.com/ossf/scorecard/blob/397c97f766bb6c2cf6740c698e52e5c74de40dbb/pkg/scorecard_result.go#L86-L89
